specification/src/main/asciidoc/webprofile/jpa-cdi/CDI-JPA.adoc

```diff
@@ -0,0 +1,120 @@
+This section specifies the requirements for Jakarta EE environments that integrate both {jpa-spec}[Persistence] and {cdi-spec}[CDI].
+
+==== Obtaining an Entity Manager using CDI
+
+A container environment might feature built-in integration of
+Jakarta Persistence with the CDI bean manager, allowing injection
+of a container-managed entity manager using the annotation
+_jakarta.inject.Inject_. In the Jakarta EE environment, the
+container must feature built-integration of Jakarta Persistence
+with the CDI bean manager.
+
+For each persistence unit, a container which features such
+built-in integration must make available a bean with:
+
+- bean type _EntityManager_,
+- the qualifiers specified by _qualifier_ XML elements in
+_persistence.xml_, or _jakarta.enterprise.inject.Default_,
+if no qualifiers are explicitly specified,
+- the scope specified by the _scope_ XML element in
+_persistence.xml_, or _jakarta.transaction.TransactionScoped_
+if no scope is explicitly specified,
+- no interceptor bindings,
+- a bean implementation which satisfies the requirements of
+this specification for a container-managed entity manager.
+
+==== Obtaining an Entity Manager Factory using CDI
+
+A container environment might feature built-in integration of
+Jakarta Persistence with the CDI bean manager, allowing
+injection of a container-managed entity manager factory using
+the annotation _jakarta.inject.Inject_. In the Jakarta EE
+environment, the container must feature built-integration of
+Jakarta Persistence with the CDI bean manager.
+
+For each persistence unit, a container which features such
+built-in integration must make available a bean with:
+
+- bean type _EntityManagerFactory_,
+- the qualifiers specified by _qualifier_ XML elements in
+_persistence.xml_, or _jakarta.enterprise.inject.Default_,
+if no qualifiers are explicitly specified,
+- scope _jakarta.enterprise.context.ApplicationScoped_,
+- bean name given by the name of the persistence unit,
+- no interceptor bindings,
+- a bean implementation which satisfies the requirements of
+this specification for a container-managed entity manager
+factory.
+
+Furthermore, the container must make available five beans with:
+
+- bean types _CriteriaBuilder_, _PersistenceUnitUtil_, _Cache_,
+_SchemaManager_, and _Metamodel_, respectively,
+- the qualifiers specified by _qualifier_ XML elements in
+_persistence.xml_, or _jakarta.enterprise.inject.Default_,
+if no qualifiers are explicitly specified,
+- scope _jakarta.enterprise.context.Dependent_,
+- no interceptor bindings,
+- a bean implementation which simply obtains the instance of
+the bean type by calling the appropriate getter method of
+the _EntityManagerFactory_ bean.
+
+==== persistence.xml Extended Schema
+The Persistence 3.1 specification schema includes an extension point that allows one to add elements for configuration of integration concerns. The required approach for declaring a CDI  scope and qualifier elements.
```

Comment:
```suggestion
The Persistence 3.2 specification schema includes an extension point that allows one to add elements for configuration of integration concerns. The required approach for declaring a CDI  scope and qualifier elements.
```

Reply:
does the explicit usage of the spec version number make sense here? The version _in action_ is mentioned somewhere at the beginning already

Reply:
I have removed it from the text, but it does still show up in the schema used in the persistence.xml example.